server/app/services/program/ProgramDefinition.java

```diff
@@ -857,6 +867,11 @@ public Builder addLocalizedDescription(Locale locale, String description) {
       return this;
     }
 
+    public Builder addLocalizedShortDescription(Locale locale, String description) {
```

Comment:
It's going to be used when I add in translations. I went back and forth about where to draw the line for what we will need for translations and landed here since I'm making all the other updates to this file now. I'm happy to take it out of this PR though. That actually might make more sense in case we need to revert anything. I talked myself into removing it lol.